.github/ISSUE_TEMPLATE/engine-crash.md

```diff
@@ -1,10 +1,9 @@
 ---
-name: Engine Crash
-about: Post crash logs with translated stack traces
-title: EngineCrash [EngineVersion] at [last sane source line]
-labels: bug
-assignees: ''
-
+name = "Engine Crash"
```

Comment:
forgot to fix the inside, I believe some search and replace broke the original format, we need to restore to previous.